Tools/CISupport/ews-build/steps.py

```diff
@@ -1506,7 +1509,7 @@ def getResultSummary(self):
         return {'step': 'Failed to add comment on bug {}'.format(self.bug_id)}
 
     def doStepIf(self, step):
-        return CURRENT_HOSTNAME == EWS_BUILD_HOSTNAME
+        return CURRENT_HOSTNAME == EWS_BUILD_HOSTNAME and bool(self.getProperty('patch_id', False))
```

Comment:
If you are disabling this step, you might want to disable CloseBug step as well.

Reply:
Reading through the code, `CloseBug` only applies to `CommitQueue` (which will never support pull-requests) while `CommentOnBug` and `SetCommitQueueMinusFlagOnPatch` support builders and testers too.

Reply:
If I remember correctly, `CommentOnBug` is run only on CommitQueue. Regular builders and tester queues do not comment on Bugzilla bug.

Reply:
Closer inspection does corroborate that, yes. I guess the comments to bugzilla was old behavior. Going to remove `CommentOnBug` from this change.

Reason I'm trying to keep this change narrow is because I would like things to break should someone try and send PRs through `CommitQueue` steps in the future, since committing PRs is a different process than committing patches.